Assets/Scripts/SpawnTower.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+public class SpawnTower : MonoBehaviour
+{
+    
+    public GameObject objToSpawn;
+    public Transform objSpawnPlace;
+    
+    private float myTime = 0F;
+    private float nextPlacement = 1F;
+    private float placementRate = 1F;
```

Comment:
Make these variables accessible from the outside? ([SerializeField])

Reply:
changed accordingly